### PR TITLE
update task version to 1.1.0

### PR DESCRIPTION
**Pull request by shahzzzam**
## Purpose

* ...
Add a simple Task that uses Managed Identities

## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: msidoc.md [hunk 1]
```diff
@@ -0,0 +1,381 @@
+# Task with Managed Identities
+
+
+## Access to Azure KeyVault and `az login` using Managed Identity
+
+Task Definition [file](https://github.com/shahzzzam/acr-tasks/blob/master/managed-identities.yaml) on GitHub.
```
Comment (northtyphoon) [inline msidoc.md]: update the link

File: msidoc.md [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+$ az acr task create -n multiple-reg -r myregistry -c https://github.com/Azure-Samples/acr-tasks.git#:multipleRegistries \
+   -f testtask.yaml --commit-trigger-enabled false --pull-request-trigger-enabled false --assign-identity
+
+{
+  "agentConfiguration": {
+    "cpu": 2
+  },
+  "creationDate": "2019-05-16T22:54:04.620200+00:00",
+  "credentials": null,
+  "id": "/subscriptions/84c559c6-30a0-417c-ba06-8a2253b388c3/resourceGroups/samashahtest-rg/providers/Microsoft.ContainerRegistry/registries/myregistry/tasks/multiple-reg",
```
Comment (dlepow) [inline msidoc.md]: You might want to remove/anonymize identifying info in return

File: msidoc.md [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+      "name": "defaultBaseimageTriggerName",
+      "status": "Enabled"
+    },
+    "sourceTriggers": null,
+    "timerTriggers": null
+  },
+  "type": "Microsoft.ContainerRegistry/registries/tasks"
+}
+
+// Capture id and principal_id from response.
```
Comment (dlepow) [inline msidoc.md]: I found I needed to assign acrpush role (for example) to each of the target registries to get this scenario to work.

File: msidoc.md [hunk 4]
```diff
@@ -0,0 +1,415 @@
...
+
+$ az acr task credential add -n multiple-reg -r myregistry --login-server customregistry1.azurecr.io --use-identity $principal_id
+{
```
Comment (dlepow) [inline msidoc.md]: To use system assigned MSI, the command I needed to use looked like (with the `[system]`:

```
az acr task credential add -n multiple-reg -r myregistry --login-server customregistry1.azurecr.io --use-identity [system]
```

File: msidoc.md (hunk 2 above)
Comment (shahzzzam) [reply msidoc.md]: whats the usual practice of anonymizing GUIDs?
There is `subscriptionId`, `principalId`, `tenantId` , etc. Should I anonymize them all with xxxxxxxxx-xxxxx-xxxx-xxxx-xxxxxxxxxxxxx ?

File: msidoc.md (hunk 3 above)
Comment (shahzzzam) [reply msidoc.md]: youre right! addressed

File: msidoc.md (hunk 4 above)
Comment (shahzzzam) [reply msidoc.md]: good catch

File: msidoc.md (hunk 2 above)
Comment (dlepow) [reply msidoc.md]: @shahabhijeet - Yes, that works

**Comment by shahzzzam** [comment]
@dlepow done

**Comment by shahzzzam** [comment]
if this looks good, @dlepow can we close this.

**Comment by dlepow** [comment]
@shahzzzam - Sam, looks good to me. I think you can just go ahead and merge. Thanks!

File: managed-identities.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    env: 
+      - MY_SECRET='{{.Secrets.name}}' 
+
+  # Build/Push the website to source registry
+  - cmd: docker build -t {{.Run.Registry}}/my-website:{{.Run.ID}} https://github.com/Azure-Samples/aci-helloworld.git
+  - push: 
+    - "{{.Run.Registry}}/my-website:{{.Run.ID}}"
+  
+  # Login to Azure and list the tags to verify if we have the Image!
+  - cmd: microsoft/azure-cli az login --identity
```
Comment (dlepow) [inline managed-identities.yaml]: @shahzzzam - Sam, this KV/login example uses a user-defined identity as explained in your doc. Is it OK for the task step to login to az CLI with a system-managed identity (presumably derived from the task?), or should it be logging in with the user-assigned identity?

Comment (shahzzzam) [reply managed-identities.yaml]: @dlepow it doesn't matter.. you can either assign a user assigned identity or a system assigned identity to the task. As long as you the assign identity a role in Azure, we will be able to login. Overall, in any example, you can use user-assigned or system-assigned identity. Hope this helps.

**Comment by shahzzzam** [comment]
@dlepow correct. These are called Aliases now. I believe we have a thread on Alias documentation.

**Comment by dlepow** [comment]
@shahzzzam - Ah, makes sense now. I'll merge this and get going with the alias update

**Pull request merged**